pkg/controller/download_request_controller.go

```diff
@@ -178,20 +180,22 @@ func (c *downloadRequestController) processNextWorkItem() bool {
 // processDownloadRequest is the default per-item sync handler. It generates a pre-signed URL for
 // a new DownloadRequest or deletes the DownloadRequest if it has expired.
 func (c *downloadRequestController) processDownloadRequest(key string) error {
-	glog.V(4).Infof("processDownloadRequest for key %q", key)
+	logContext := c.logger.WithField("key", key)
+
+	logContext.Info("Running processDownloadRequest")
 	ns, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		glog.V(4).Infof("error splitting key %q: %v", key, err)
+		logContext.WithField("error", err).Info("Error splitting key")
 		return err
 	}
 
 	downloadRequest, err := c.downloadRequestLister.DownloadRequests(ns).Get(name)
 	if apierrors.IsNotFound(err) {
-		glog.V(4).Infof("unable to find DownloadRequest %q", key)
+		logContext.Info("Unable to find DownloadRequest")
 		return nil
 	}
 	if err != nil {
-		glog.V(4).Infof("error getting DownloadRequest %q: %v", key, err)
+		logContext.WithField("error", err).Info("Error getting DownloadRequest")
```

Comment:
Please don't log an error at Info level, please don't log and return an error, just return it.